src/pytorch_lightning/callbacks/pruning.py

```diff
@@ -261,7 +261,7 @@ def _wrap_pruning_fn(pruning_fn: Callable, **kwargs: Any) -> Callable:
     def make_pruning_permanent(self, module: nn.Module) -> None:
         """Removes pruning buffers from any pruned modules.
 
-        Adapted from https://github.com/pytorch/pytorch/blob/1.7.1/torch/nn/utils/prune.py#L1176-L1180
+        Adapted from https://github.com/pytorch/pytorch/blob/master/torch/nn/utils/prune.py#L1172-L1200
```

Comment:
The specific version here is desired, as it was the exact code that was adapted back then. Otherwise, if pointing to master, it gets out of date.

Reply:
Right, but what would be the solution if they have deleted the 1.7.1 branch from their repository?

Reply:
The branch is gone but the tag is still there. The difference is subtle: v1.7.1 vs 1.7.1 (the "v"):

https://github.com/pytorch/pytorch/blob/v1.7.1/torch/nn/utils/prune.py#L1176-L1180

Perhaps the code lines need to be adjusted

Reply:
That makes sense. Is ths better
https://github.com/pytorch/pytorch/blob/v1.7.1/torch/nn/utils/prune.py#L1118-L1122

Reply:
Resolved this.